src/abstractions/Time.cs

```diff
@@ -26,6 +27,29 @@ public struct Time
         /// <returns>A new <see cref="TimeOnly"/> structure whose hours, minutes, seconds and milliseconds are equal to those of the supplied time.</returns>
         public static implicit operator TimeOnly(Time time) => new(time.DateTime.Hour, time.DateTime.Minute, time.DateTime.Second, time.DateTime.Millisecond);
 #endif
+        /// <summary>
+        /// Indicates whether the current object is equal to another object of the same type.
+        /// </summary>
+        /// <param name="other">An object to compare with this object.</param>
+        /// <returns>true if the current object is equal to the other parameter; otherwise, false.</returns>
+        public bool Equals(Time other) => (Hour, Minute, Second) == (other.Hour, other.Minute, other.Second);
+
+        /// <inheritdoc />
+        public override bool Equals(object? o) => (o is Time other) && Equals(other);
+
+        /// <inheritdoc />
+        public override int GetHashCode()
+        {
+#if NET6_0_OR_GREATER
+            return HashCode.Combine(Hour, Minute, Second);
+#else
+            int hash = 17;
+            hash = hash * 23 + Hour.GetHashCode();
+            hash = hash * 23 + Minute.GetHashCode();
+            hash = hash * 23 + Second.GetHashCode();
```

Comment:
should the exponents here be different from one another?

Reply:
Exponents can be the same

Reply:
I'm okay leaving it as is unless @baywet has any concerns here.

Reply:
I just realized that you multiply the previous hash value by the exponent.
 This is not usually how I do it. I usually multiply the new value by the new exponent. 
I guess in that case the math works if you're using the same exponent.
I don't have any more reservations. Thank you for the discussion on this topic. 
Resolving.